extensions/amp-date-picker/0.1/date-picker-common.js

```diff
@@ -53,7 +53,7 @@ export function withDatePickerCommon(WrappedComponent) {
    */
   function isOutsideRange(min, max, date) {
     const maxInclusive = max && moment(max);
-    const minInclusive = min && moment(min);
+    const minInclusive = min ? moment(min) : getDefaultMinDate(max);
```

Comment:
shouldn't this be `getDefaultMinDate(min)`?

Reply:
No, but the method is confusing. The default minimum date depends on what the maximum date is. The parameter is named max in the function to help clarify 

https://github.com/ampproject/amphtml/blob/611c438f978c16784f73d10ef67d29cf158fcbb6/extensions/amp-date-picker/0.1/date-picker-common.js#L39